src/pilot/dell_nfv.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/python
+
+# Copyright (c) 2016-2018 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+
+import cpu_siblings
+import os
+import sys
+import time
+import subprocess
+import string
+import logging
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from novaclient.v2 import aggregates
+from novaclient.v2 import hosts
+from novaclient.v2 import servers
+from credential_helper import CredentialHelper
+from datetime import datetime
+import novaclient.client as nova_client
+from novaclient import client as nvclient
+from ironic_helper import IronicHelper
+from dracclient import client
+logging.basicConfig()
+logger = logging.getLogger(os.path.splitext(os.path.basename(sys.argv[0]))[0])
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+home_dir = os.path.expanduser('~')
```

Comment:
This is required to get environment files from home path.

Reply:
I think we have to break down the stack creation by using the heat-api client instead of using the heat client. By choosing this way we don't need to use stackrc file any longer.